sites/fast-website/src/app/components/fast-frame/fast-frame.ts

```diff
@@ -141,20 +148,25 @@ export class FastFrame extends FASTElement {
         this.accentPalette = createColorPalette(accentRGB);
     }
 
-    public themeChange = (e: CustomEvent): void => {
+    private updateBackgroundColor(): void {
```

Comment:
@bheston I might be missing it, but with the default value gone, should this be initially invoked during connected callback?

Reply:
I wasn't entirely sure how the lifecycle worked, but I put the call in the constructor and it seemed to come up fine. I'm not sure how that works if used as an attribute though, or if it matters if it's defaulted or not.

Reply:
@bheston can we move to connectedCallback() to be safe here?